include/glow/ExecutionEngine/ExecutionEngine.h

```diff
@@ -55,6 +63,9 @@ class ExecutionEngine final {
   /// Set the code generator to a custom \p backend.
   void setBackend(Backend *backend);
 
+  /// Set the code generator to use a custom \p backend without owning it.
+  void useBackend(Backend *backend);
```

Comment:
Can you keep the old version of setBackend and introduce a param on whether backend is owned or not?

Reply:
Sure